packages/eas-cli/src/build/configure.ts

```diff
@@ -0,0 +1,82 @@
+import { getConfig } from '@expo/config';
+import fs from 'fs-extra';
+import path from 'path';
+
+import log from '../log';
+import { ensureLoggedInAsync } from '../user/actions';
+import { gitAddAsync } from '../utils/git';
+import { configureAndroidAsync } from './android/configure';
+import { ConfigureContext } from './context';
+import { configureIosAsync } from './ios/configure';
+import { BuildCommandPlatform } from './types';
+import {
+  ensureGitRepoExistsAsync,
+  ensureGitStatusIsCleanAsync,
+  modifyAndCommitAsync,
+} from './utils/repository';
+
+export async function configureBuildAsync(options: {
```

Comment:
```suggestion
export async function configureAsync(options: {
```
or
```suggestion
export async function configureProjectForEASBuildAsync(options: {
```
or sth similar.
Please make the decision on your own :)